.kokoro-windows/Initialize-GoogleProjectForTesting.ps1

```diff
@@ -98,6 +98,7 @@ roles/monitoring.admin
 roles/pubsub.admin
 roles/resourcemanager.projectIamAdmin
 roles/storage.admin
+roles/viewer

```

Comment:
This role is too broad.  Please chose a role with more limited privileges.

Reply:
Replaced it with cloudasset.viewer.